infrastructure/variables.tf

```diff
@@ -43,3 +43,8 @@ variable "subscription" {}
 variable "common_tags" {
   type = "map"
 }
+
+variable "team_contact" {
+  default     = "#platform-engineering"
```

Comment:
use #rpe, this should be a place people can come to contact us, platform-engineering is private